assets/extensions/main.js

```diff
@@ -38,28 +35,66 @@ const Main = () => {
 		);
 	}
 
+	const freeExtensions = extensions.filter(
+		( extension ) => extension.price === 0
+	);
+	// TODO: Specify which plugins are third party
+	const thirdPartyExtensions = extensions.filter(
```

Comment:
Not for this PR, but just as a note. We could expose in the senseilms.com API the product categories because the third-party plugins are identified as a category.